opencontainers/struct.py

```diff
@@ -42,7 +55,10 @@ def __init__(
         hide=False,
     ):
         self.name = name
-        self.value = value
+        if type(attType) is list and value is None:
```

Comment:
```suggestion
        if isinstance(attType, list) and value is None:
```

Reply:
You could make this one line if you like, although it might not be best for readability (up to you)
```
self.value = [] if isinstance(attType, list) and value is None else value
```
(or the reverse of that)